Userland/Libraries/LibAudio/MP3Types.h

```diff
@@ -58,6 +58,7 @@ struct Header {
     bool original_bit;
     Emphasis emphasis;
     u16 crc16;
+    size_t header_size;
```

Comment:
We should really zero-initialize all of the plain types in this struct. Not doing so has caused UB in the past.

Reply:
Right, I'll throw in a commit to fix the existing fields.